docs/ops-recipe/harborPodPhaseStatus.md

```diff
@@ -0,0 +1,86 @@
+---
+
+title: "harborPodPhaseStatus"
+
+---
+
+This alert fires because one or more Harbor pods are in a failing, pending or unknown [phase](https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/). This could occur during deployment or at any point during Harbors lifecycle. 
+
+
+**Harbor exporter**
+
+This is a specific case for when a disruption occurs with the `harbor-exporter` pod, which might lead to issues with exporting metrics to Prometheus, and connecting with Postgres.
+
+Likely you wont see this in the terminal and instead will receive the notification through prometheus. Restart the harbor-exporter pod and check the logs.
+
+```
+
+kubectl -n harbor-cluster delete pod harbor-exporter
+
+kubectl -n harbor-cluster logs harbor-exporter
+
+```
+
+The error is likely to do with the connection to the exporter and other components being down. Try restarting the service. 
+
+```
+
+kubectl -n harbor-cluster delete service harbor-exporter
+
+```
+
+If this doesn't work and you have explored other options a likely fix is to re-deploy harbor itself.
+
+**General**
+
+In general if your pods are in a failed, pending or unknown phase you should double check your configurations and the state you expect the pods to be in.
+
+Check what the logs for the pod says:
+
+```
+
+kubectl -n harbor-cluster logs harbor-$COMPONENT
+
+```
+
+You can also describe the pod:
+
+```
+
+kubectl -n harbor-cluster describe pod harbor-$COMPONENT
+
+```
+
+Based on what the logs say try to figure out what is causing the error. Usually it will be the result of an improper configuration. Most configuration for the cluster is created in the [fullstack.yaml](https://github.com/goharbor/harbor-operator/blob/master/manifests/samples/full_stack.yaml). Below are some things to look out for if you cannot find the cause.
+
+**Secrets**
```

Comment:
For the section below, same comments as for the 1st recipe, as it seems to be copy-paste